Nef_3/include/CGAL/Nef_3/SNC_k3_tree_traits.h

```diff
@@ -56,15 +57,15 @@ class ComparePoints {
 
 
 template <typename Coordinate, typename EK>
-class ComparePoints<CGAL::Lazy_kernel<EK>, Coordinate> {
+class Compare_points<CGAL::Lazy_kernel<EK>, Coordinate> {
```

Comment:
@afabri I think this template specialisation (and also for Smaller_than) should be removed because:

a) It doesn't seem to match against Epeck kernels.
b) I think CGAL::compare_x already does an approx comparison when possible.

Reply:
I agree. I am even wondering if we should not add the 3D equivalent to  [`Compare_x_2`](https://github.com/CGAL/cgal/blob/master/Filtered_kernel/include/CGAL/internal/Static_filters/Compare_x_2.h).

Reply:
I just convinced myself that there is no need for the "3D equivalent"   as there is a typedef in Static_filters.h

Reply:
I was wondering why a `typedef` as sufficient for `Compare_x_3` but a dedicated functor was required for `Compare_x_2`. The difference is:
  - the concept [`CompareX_3`](https://doc.cgal.org/latest/Kernel_23/classKernel_1_1CompareX__3.html) specifies only one overload, that only compares coordinates,
  - whereas the concept [`CompareX_2`](https://doc.cgal.org/latest/Kernel_23/classKernel_1_1CompareX__2.html) has overloads that require comparison of compute values.